qiskit/compiler/schedule.py

```diff
@@ -25,7 +25,7 @@
 from qiskit.exceptions import QiskitError
 from qiskit.pulse import InstructionScheduleMap, Schedule
 from qiskit.providers import BaseBackend
-from qiskit.scheduler import schedule_circuit, ScheduleConfig
```

Comment:
Leaving a note re: our discussion, to revert this for now (maybe make an issue for discussion)

Reply:
reverted in b7ce9211a958a0821510f1541